swagger.yml

```diff
@@ -1024,11 +1046,7 @@ components:
           nullable: true
     TechnologyStatus:
       type: object
-      required:
-        - Product
-        - HostsCount
-        - Compliance
-        - UnpaidDues
+      title: ''
```

Comment:
title: ''?????

Reply:
with Stoplight it is not visible. removed manually